metadata/layout.conf

```diff
@@ -11,3 +11,7 @@ sign-manifests = false
 
 # Let's start with new EAPIs if possible
 eapis-banned = 0 1 2 3 4 5
+
+# Part of ability to mask features in overlays
+# https://trofi.github.io/posts/201-masking-a-package-in-gentoo-overlay.html
+profile-formats = portage-2
```

Comment:
I can't find any documentation about this on the wiki: https://wiki.gentoo.org/wiki/Repository_format/metadata/layout.conf

Are you sure this is required? Considering that we already have sys-apps/portage-3.X.Y in the tree, this feature might already be integrated into portage by default. And thus this might be unnecessary.

Reply:
Also, pkgcheck says `UnusedProfileDirs: unused profile dirs: 'features', 'features/selinux'`, so I wonder if this is doing anything at all.

Reply:
In regards to `UnusedProfileDirs`, `features/selinux/package.mask` must be doing something as without it I get the following:

```
  dependency.bad [fatal]        5
   sys-block/systemd-swap/systemd-swap-4.4.0.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.1/hardened/selinux)
[     'dev-python/python-systemd[python_targets_python3_8(-),-python_single_target_python3_8(-)]']
   sys-block/systemd-swap/systemd-swap-4.4.0.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.1/no-multilib/hardened/selinux)
[     'dev-python/python-systemd[python_targets_python3_8(-),-python_single_target_python3_8(-)]']
   sys-block/systemd-swap/systemd-swap-4.4.0.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.1/selinux)
[     'dev-python/python-systemd[python_targets_python3_8(-),-python_single_target_python3_8(-)]']
   sys-block/systemd-swap/systemd-swap-4.4.0.ebuild: RDEPEND: ~x86(default/linux/x86/17.0/hardened/selinux)
[     'dev-python/python-systemd[python_targets_python3_8(-),-python_single_target_python3_8(-)]']
   sys-block/systemd-swap/systemd-swap-4.4.0.ebuild: RDEPEND: ~x86(default/linux/x86/17.0/selinux)
[     'dev-python/python-systemd[python_targets_python3_8(-),-python_single_target_python3_8(-)]']
```

I would say that the pkgcheck warning is displaying as overlays don't support feature masking without this hacky stuff.

---

Without `profile-formats = portage-2` I get the following issues from repoman:

```
!!! Unable to parse profile: '/home/telans/src/guru/profiles/default/linux/amd64/17.1/hardened/selinux'
!!! ParseError: Parent '/home/telans/src/guru/profiles/default/linux/amd64/17.1/hardened/selinux/gentoo:default/linux/amd64/17.1/hardened/selinux' not found: '/home/telans/src/guru/profiles/default/linux/amd64/17.1/hardened/selinux/parent'
!!! Unable to parse profile: '/home/telans/src/guru/profiles/default/linux/amd64/17.1/no-multilib/hardened/selinux'
!!! ParseError: Parent '/home/telans/src/guru/profiles/default/linux/amd64/17.1/no-multilib/hardened/selinux/gentoo:default/linux/amd64/17.1/no-multilib/hardened/selinux' not found: '/home/telans/src/guru/profiles/default/linux/amd64/17.1/no-multilib/hardened/selinux/parent'
!!! Unable to parse profile: '/home/telans/src/guru/profiles/default/linux/amd64/17.1/selinux'
!!! ParseError: Parent '/home/telans/src/guru/profiles/default/linux/amd64/17.1/selinux/gentoo:default/linux/amd64/17.1/selinux' not found: '/home/telans/src/guru/profiles/default/linux/amd64/17.1/selinux/parent'
!!! Unable to parse profile: '/home/telans/src/guru/profiles/default/linux/x86/17.0/hardened/selinux'
!!! ParseError: Parent '/home/telans/src/guru/profiles/default/linux/x86/17.0/hardened/selinux/gentoo:default/linux/x86/17.0/hardened/selinux' not found: '/home/telans/src/guru/profiles/default/linux/x86/17.0/hardened/selinux/parent'
!!! Unable to parse profile: '/home/telans/src/guru/profiles/default/linux/x86/17.0/selinux'
!!! ParseError: Parent '/home/telans/src/guru/profiles/default/linux/x86/17.0/selinux/gentoo:default/linux/x86/17.0/selinux' not found: '/home/telans/src/guru/profiles/default/linux/x86/17.0/selinux/parent'
```